src/sidebar/test/annotation-metadata-test.js

```diff
@@ -247,6 +247,35 @@ describe('annotation-metadata', function () {
     });
   });
 
+  describe('.isPublic', function () {
+    it('returns true if an annotation is public within a group', function () {
```

Comment:
Hmm, I think this use of the term "public" might cause some confusion.

An annotation can be _private_ or _shared_ within a group. If the annotation is _shared_ with a group and the group is world-readable, then it is _public_. Does that make sense?

In that case, I would suggest we just change the description to "... is _shared_ within a group".